Makefile

```diff
@@ -93,6 +98,9 @@ NPROCS := $(shell getconf _NPROCESSORS_ONLN)
 MFLAGS := -j $(NPROCS) --warn-undefined-variables \
                        --no-print-directory --no-keep-going
 
+# Benchmark options
+BENCHMARK_EX := petsc-bp1
```

Comment:
What about handling this more like tests, where there are `run-*` targets for each test?

Reply:
I'm investigating. I didn't introduce the other three changes you commented on, so I'll leave that to @v-dobrev to reply.

Reply:
Yes, exactly.

Reply:
@jeremylt, if you are also OK with the above proposal, I can add to the makefile.

What about the `benchmarks` target? What is everyone's preference: should we keep it shorter and make it depend just on `bench-petsc-bp1` or should we be more comprehensive and have it run `$(allbechmarks)`?

Reply:
I just made the above changes and updated README.md.